presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcSplitManager.java

```diff
@@ -41,6 +41,7 @@ public ConnectorSplitSource getSplits(
             ConnectorTableLayoutHandle layout,
             SplitSchedulingContext splitSchedulingContext)
     {
+        // TODO: use translatedSql from tableLayoutHandle
```

Comment:
It's worth implementing the actual pushdown by having the jdbcSql in `JdbcSplit.additionalPredicate`. Make sure to write e2e test to verify it works.